core/src/main/java/io/aiven/klaw/service/ClusterApiService.java

```diff
@@ -127,8 +125,11 @@ public ClusterApiService(ManageDatabase manageDatabase) {
     this.manageDatabase = manageDatabase;
   }
 
-  private RestTemplate getRestTemplate() {
-    if (clusterConnUrl.toLowerCase().startsWith("https")) {
+  private RestTemplate getRestTemplate(String clusterConnectivityUrl) {
+    if (clusterConnectivityUrl == null) {
```

Comment:
Maybe check StringUtils.isEmpty(clusterConnectivityUrl) that will check if it is null or empty